data/brands/amenity/fast_food.json

```diff
@@ -6713,7 +6713,9 @@
         "brand:wikidata": "Q18636431",
         "cuisine": "mexican",
         "name": "Zambrero",
-        "takeaway": "yes"
+        "takeaway": "yes",
+        "diet:vegetarian": "yes",
```

Comment:
Hello, @Georift ! Before add vegeratian and vegan tag, maybe you can view #4855 before continue, because those tag is difficult to define what is available.

Reply:
Thanks for the link I'll have a dive. Apologies if this isn't relevant I noticed at least one other PR adding a vegan tag so I thought it was far game. Let me dig into that post and get back to this.

Reply:
Alright I guess there isn't really an appetite for having these types of tags added for restaurants / brands who aren't `vegetarian=only` or `vegan=only`.

Appreciate the quick link. I'll move over to these for some discussion.